src/main/java/hudson/plugins/jira/JiraRestService.java

```diff
@@ -275,8 +277,15 @@ public BasicIssue createIssue(String projectKey, String description, String assi
             builder.setPriorityId(priorityId);
         }
 
-        if (!assignee.equals(""))
-            builder.setAssigneeName(assignee);
+        if (!assignee.equals("")) {
```

Comment:
please use StringUtils.isBlank here to avoid NPE

Reply:
Fixed